### DISCUSSION TITLE
Beta 0.8.5

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#622] [General]
Hello, it's been awhile Roberto (:
I'm just wondering about any developments on the status of the upcoming major update.

Not to pressure you to finish faster (don't rush it) but just wanting to know how is everything going.

**Comment by dnestico** [comment]
Maybe this might help you but I could be wrong.

https://github.com/robertodoering/harpy/pull/587

**Comment by robertodoering** [comment]
The next update (#587) is about ~90% done - it's a complete technical rewrite of the app and will include a bunch of improvements and bug fixes.

This has been in the works for a while now but it'll serve as a good foundation to add future features with less time in between as well :)

**Reply by FerasK0** [reply]
oh, so it was reworking the app's foundation, makes sense now.
Great decision.

**Comment by dontobi** [comment]
That sounds very good. Then I hope that the remaining ~10% will be finished in the next days/weeks. Until then I use Twitter in the browser (Mentions) and the current Harpy Pro version.

**Comment by scottiefalkon** [comment]
Not rushing anything. How far out is 0.8.5. Ready to use my favorite Twitter app again. Thanks for all your work.
